packages/storybook/src/ui/organisms/questionnaire-table/questionnaire-table.test.tsx

```diff
@@ -200,7 +208,7 @@ describe('Questionnaire Table Methods', () => {
     expect(JSON.stringify(result)).toEqual(JSON.stringify(expectedResultOfMappingWithHeadersOnXAxis))
   })
 
-  it('Maps questionnaires data with headers along the Y axis', () => {
+  it.only('Maps questionnaires data with headers along the Y axis', () => {
```

Comment:
I accidentally left a `.only` in on my last PR that I forgot to remove - it's since been corrected. There's no need to add it to every test here :)